tutorials/getting_started/rules_scenes.md

```diff
@@ -0,0 +1,119 @@
+| Description | describes what the rule does | do not skip, provide information about what the rule does and what it does it to; leave breadcrumbs you can use later to associated rules with Items and Channels                                                                                |
+| Tags        | arbitrary tags               | Another place to leave breadcrumbs or to group rules together. Note the `Script` and `Schedule` tags have a special meaning (see previous page), and although Semantic tags are available, rules are not yet integrated into the Semantic Model. |
+
+![scenes UI rule heading](images/scenes-rule-heading.png)
+![scenes UI rule heading tags](images/scenes-rule-heading-tags.png)
+
+Here is how this rule appears in the list of rules (note the use a couple of tags):
+
+![scenes rule list entry](images/scenes-rule-list-entry.png)
+
+It also shows the status of the rule: I
+
+All but the Unique ID can be changed later.
+
+### Configuration (Actions)
+
+Finally, we want this rule to send some commands our items which is very convenient to configure in this editor.
+First, it is recommended you put all of your items in to the state that you want to store as scene.
+
+After your scene of all the devices in your room (or house) has been set up, you assign these in the following section:
+
+![empty scene configuration](images/scene-configuration-empty.png)
+
+Choose the item either by clicking on ![item tree lookup](images/item-tree-lookup.png) to select it via the semantic model or on ![item list lookup](images/item-list-lookup.png) via an item list.
+Note that you can select multiple items at the same time.
+
+The item will then appear in the list together with the current state shown next to it.
+This state will later be sent when the rule is executed.
+
+![scene-entry1](images/scene-entry1.png)
+
+In the above case the effect "Colored Worm" would be sent to the light panels.
+If you are not happy with that state either overwrite it manually or set the item to the state you want and the press the arrow to refresh to the current state:
+
+![scene-item-refresh](images/scene-item-refresh.png)
+
+The ![play-button](images/play-button.png) allows you to test the item state value that is currently chosen.
+
+Add as many items until you end up with the scene that you wish to create:
+
+![complete-scene](images/scene-complete.png)
+
+You can reorder the items via the ![scene-reorder](images/scene-reorder-button.png) button.
+This also defines the sequence within which the commands are sent to the device's items.
+
+Similarly to all other rules play button ![scene-pause-and-play](images/pause_and_play.png) allows you to run the rule and test it out that way and the pause button allows you to halt the rule in case you want to prevent it to be executed.
```

Comment:
```suggestion
Similarly to all other rules the play button ![scene-pause-and-play](images/pause_and_play.png) allows you to run the rule and test it out that way and the pause button allows you to halt the rule in case you want to prevent it to be executed.
```